EIPS/eip-7488.md

```diff
@@ -0,0 +1,51 @@
+
+## Motivation
+
+Since its humble beginnings, the ethereum project has been open to the public and seen many contributions. The past years, however, have made it clear that such an open ecosystem design leads to a wide array of problems such as ongoing consensus problems, increasingly high transaction costs, cases of fraud, false ownership claims of assets such as the DAO hack and resulting fork (see [EIP-779](./eip-779.md)), illegal contracts and privacy intrusion, data theft, artifical and malicious content, and many more such things. 
+
+Halting the IPFS and Swarm storage process, which enabled and multiplied above problems, as well as historically drained the ethereum ecosystem and infrastructure of important ressources and instead developing use cases for the original community would be a remedy for almost all of these problems.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+Mainnet (ID: 1) MUST be switched to permissioned. The permissioned ethereum blockchain then either applies proof-of-authority consensus (see [EIP-225](./eip-225.md)), or alternatively, a new consensus mechanism. In part, this may be achieved by halting IPFS, Swarm, and Whisper, after a successfull testnet version has been deployed, for which Goerli or Sepolia may provide blue prints to some degree.
+
+Before halting IPFS and Swarm, packages SHOULD be transfered via p2p to the original owner, including the original data.
+
+Since for some existing dApps, token holdings would likely be exposed to such a protocol change, it is REQUIRED to decide what happens to affected [ERC-721](./eip-721.md) tokens and [ERC-20](./eip-20.md) coins. A potential solution would be to gather all tokens and decide step-by-step which ones would still be valid, could be transfered and/or sold to benefit the community, and which ones should be burned.
+
+## Rationale
+
+Alternatively, if this EIP "Ground Zero" should fail and no similar solution will be found, a temporary halt of the ethereum blockchain may be considered instead.
```

Comment:
This is not what the Rationale section is for. The Rationale section is for discussing design decisions. In particular--why is PoA the best solution to the problems described in the Motivation? Why does IPFS, a completely unrelated technology, have to be "stopped"? Why should the new validators go through all the assets and decide whose to discard?

All of these questions should be answered for me to consider this Rationale section complete.